include/deal.II/lac/arpack_solver.h

```diff
@@ -199,38 +225,59 @@ class ArpackSolver : public Subscriptor
   /**
    * Exceptions.
    */
-  DeclException2 (ExcInvalidNumberofEigenvalues, int, int,
+  DeclException2 (ArpackExcInvalidNumberofEigenvalues, int, int,
                   << "Number of wanted eigenvalues " << arg1
                   << " is larger that the size of the matrix " << arg2);
 
-  DeclException2 (ExcInvalidNumberofArnoldiVectors, int, int,
+  DeclException2 (ArpackExcInvalidEigenvectorSize, int, int,
+                  << "Number of wanted eigenvalues " << arg1
+                  << " is larger that the size of eigenvectors " << arg2);
+
+  DeclException2 (ArpackExcInvalidEigenvectorSizeNonsymmetric, int, int,
+                  << "To store all the real and some additional complex columns for " << arg1
+                  << "eigenvectors, its size equal to " << arg2
```

Comment:
"To store the real and complex parts of 4 eigenvectors in real-valued vectors, their size (currently set to 4) should be greater than or equal to 5."